aten/src/ATen/Dispatch.h

```diff
@@ -98,6 +98,7 @@ TORCH_API void record_kernel_function_dtype(std::string name);
     return __VA_ARGS__();                                                   \
   }
 
+namespace at {
```

Comment:
nit: use concatenated namespaces. We are using C++17 now after alll

Reply:
(won't fix) Thanks for the comment @Skylion007. I changed from concatenated namespaces back to current form to be consistent with the rest of the file.

Reply:
This includes a bit involved patch internally, imo we should split it in a few patches, one that move those to `at::detail` namespace but keeps old macros there for BC pruposes...